src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#home">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="#"
+          class="icon"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#samsung">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav_list">
          <li class="nav_item">
            <a class="nav_link is-active" href="#apple">Apple</a>
          </li>
          
          <li class="nav_item">
            <a class="nav_link" href="#samsung">Samsung</a>
          </li>
```
add empty lines between multiline elements